daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/compiler/Compiler.scala

```diff
@@ -67,6 +67,7 @@ import edu.illinois.ncsa.daffodil.processors.parsers.NotParsableParser
 import edu.illinois.ncsa.daffodil.processors.unparsers.NotUnparsableUnparser
```

Comment:
Just using to try the "start a review" button, which is grey until you type a comment.